specs/network_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,115 @@
+= Specification for adding network details in node hardware inventory
+
+Add network details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently network details are missing in node hardware inventory.
+we have to add network details like IPv4, IPv6, netmask as a part of node
+hardware inventory.
+
+== Use Cases
+
+Tendrl should show the current status of active network interface.
+
+== Proposed change
+* Netifaces package is used to find active network interface in particular
+  node.
+* Add netifaces in requirements.txt and tendrl-node-agent.spec as a dependency.
+* Modify pull_hardware_inventory.py to find active network interfaces details
+  except lo, virbr0, virbr0-nic and update those details into node_inventory
+  list.
+* Create a new class called Network in persistence with respective attribute.
+* Modify manager.py to populate network details from node_inventory list to
+  Network object.
+* Modify persister.py to persist the Network object in etcd.
+
+== Alternatives
+
+None
+
+== Data model impact
+
+* Adding network details in node hardware inventory
+
+== Impacted modules
+
+=== Tendrl/common impact
+
+None
+
+=== Tendrl/node_agent impact
+
+* tendrl/node_agent/manager/manager.py
+* tendrl/node_agent/manager/pull_hardware_inventory.py
+* tendrl/node_agent/persistence/persister.py
+* requirements.txt
+* tendrl-node-agent.spec
+
+=== Tendrl/gluster_integration impact
+
+None
+
+=== Tendrl/ceph_integration impact
+
+None
+
+
+== Security impact
+
+None
+
+== Notifications/Monitoring impact
```

Comment:
I reviewed the alerting spec. It does not answer my question. How are we tracking changes to the hardware inventory? We may need a short and a long term solution to this problem. @anmolbabu @nthomas-redhat